### PR TITLE
Add GitHub events sample data

### PR DESCRIPTION
**Pull request by owenthereal**
Add GitHub events sample data

### CODE REVIEW

File: getting-started/loading-sample-data.md [hunk 1]
```diff
@@ -1,2 +1,18 @@
...
+You can download it [here](../../.gitbook/assets/github_events.sql) and load it into Hydra:
+
+```console
+psql ... -f path_to_github_events.sql
+```
+
+or in `psql` console:
+
+```
+\d path_to_github_events.sql
```
Comment (wuputah) [inline getting-started/loading-sample-data.md]:
```suggestion
\i path_to_github_events.sql
```

File: getting-started/loading-sample-data.md [hunk 2]
```diff
@@ -1,2 +1,18 @@
 # Tutorial and Sample Data
 
+This section contains sample dataset that you can load into Hydra.
+
+## GitHub Events Dataset
+
+We prepared a dataset from the [GH Archive](https://www.gharchive.org/) that contains 9MB of events on `2015-01-01-15` that have 4000 rows.
+You can download it [here](../../.gitbook/assets/github_events.sql) and load it into Hydra:
```
Comment (wuputah) [inline getting-started/loading-sample-data.md]: Hmm, dont love it. Is there anywhere else we can stick this file?

Comment (owenthereal) [reply getting-started/loading-sample-data.md]: We can move to another GitHub repo

Comment (wuputah) [reply getting-started/loading-sample-data.md]: S3?

Comment (wuputah) [reply getting-started/loading-sample-data.md]: Could also gzip it, probably compresses by 90%

Comment (owenthereal) [reply getting-started/loading-sample-data.md]: I moved it to S3: https://github.com/HydrasDB/docs/pull/19/commits/a71e2ec4a4b4c2f4a8502fac1bd56ec86a8e425c

**Pull request merged**